generation/new_client_hermetic_build/README.md

```diff
@@ -0,0 +1,160 @@
+# New client generation (GitHub Action)
+This new generation workflow enables generation of new libraries by
+ - appending a new library to our [generation_config.yaml](https://github.com/googleapis/google-cloud-java/blob/c7429c0eec419c01d4e2fe14d063b9335efb810b/generation_config.yaml).
```

Comment:
Maybe use numeric order is better since these are 3 steps in the workflow.

Reply:
Done